develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-themes-generator.markdown

```diff
@@ -76,19 +76,16 @@ You've completed installing the Liferay Theme Generator's dependencies.
 
 Now you're ready to install the Liferay Theme Generator. 7.x.x versions of the 
 Theme Generator let you create themes for 6.2 and 7.0. Versions 8.x.x and up 
-let you create themes for later versions of @product@. Install version 7.2.0 
-globally by executing this command:
+let you create themes for later versions of @product@.
 
-    npm install -g generator-liferay-theme@7.2.0
-    
-+$$$
-
-**Note:** Version 7.2.0 is the latest 7.x.x Theme Generator version at the time 
-of this writing. Always use the latest 7.x.x version to build with the most 
-recent fixes and updates.
-
-$$$
+To create themes for **6.2** and **7.0**:
 
+    npm install -g generator-liferay-theme@^7.x.x
+    
+To create themes for **7.0** / **7.1** / later versions of @product@:
```

Comment:
@marcoscv-work shouldn't this be 7.1 and later versions of the product? Version 7.x.x is for 7.0 right?

Reply:
Oh yes, it is a little bit strange but both versions can create 7.0 themes, this is 8.x.x:

![image](https://user-images.githubusercontent.com/7963804/48566338-d9e60a80-e8fa-11e8-91a6-d4ffcdf393c8.png)

and this: 7.x.x

![image](https://user-images.githubusercontent.com/7963804/48566435-13b71100-e8fb-11e8-8035-9bb1769c89dd.png)

Reply:
Ah, ok. That is a bit odd. Alright. Thanks!